test/unit/specs/components/Personalization/actions/insertAfter.spec.js

```diff
@@ -15,7 +15,7 @@ describe("Personalization::actions::insertAfter", () => {
     cleanUpDomChanges("insertAfter");
   });
 
-  it("should inser after personalized content", () => {
+  it("should inser after personalized content", done => {
```

Comment:
Typo: `should inser`. Also, instead of dealing with the `done` arg, I believe you can just return the promise from your promise chain, remove the calls to `done()`, and remove your `.catch` entirely. Jasmine will consider your test done when the promise chain resolves. If the promise chain ends up rejected, it will consider the test failed, which I think is what you're going after here.

Reply:
Thanks for the tip. I wasn't aware that Jasmine can handle promises via `return`, now it is much cleaner.

Fixed the typo as well.